run_tests.py

```diff
@@ -1,3 +1,4 @@
+from io import open
 import optparse
```

Comment:
We're pretty consistent here about importing modules and not things from modules. What do you think about just importiong `io` below and using

```py
with io.open(...) as fd:
```

Reply:
Done!